www-servers/mongrel2/mongrel2-1.11.0-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="A language agnostic web server focused on web applications"
+HOMEPAGE="http://mongrel2.org"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="net-libs/zeromq
+	>=net-libs/mbedtls-2.1[havege]
+	dev-db/sqlite:3"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	cp "${FILESDIR}/systemtls.mak" Makefile || die
+	epatch "${FILESDIR}/${P}-polarssl-platform-590512.patch"
```

Comment:
in EAPI 6, you can use eapply instead of epatch and therefore get rid of eutils.